app/components/shared/previews/notification_component_preview/preview.html.slim

```diff
@@ -0,0 +1,17 @@
+= form_for @preview_form, url: preview_view_component_path("#{@preview.preview_name}/#{@example_name}"), html: { method: "get" } do |f|
```

Comment:
Does this `f` get used before the last line of this file? If not we can move this line down there.